test/system/current_user_test.rb

```diff
@@ -56,11 +56,13 @@ class CurrentUserTest < ApplicationSystemTestCase
     assert_text '分報チャンネル URLはDiscordのチャンネルURLを入力してください'
   end
 
-  test 'Do not show after graduation hope when advisor' do
+  test 'Do not show after graduation hope when advisor or mentor' do
```

Comment:
他のテストに合わせて最初は小文字でいいと思います〜

```suggestion
  test 'do not show after graduation hope when advisor or mentor' do
```

Reply:
@komagata 
2de718b5aee6e1da64b61d6b50c754693c482e44
こちらで修正しました〜